unblob/processing.py

```diff
@@ -42,6 +36,8 @@ class ExtractionConfig:
     max_depth: int = DEFAULT_DEPTH
     process_num: int = DEFAULT_PROCESS_NUM
     keep_extracted_chunks: bool = False
+    eliminate_whole_file_chunks: bool = True
```

Comment:
I guess, we should reverse the meaning of this flag to match with the flag above. E.g. `keep_whole_file_chunks = False`

Reply:
this is a temporary flag, needed for the tests only - it is reverted in the next commit, after the test data has changed (which is a huge change, did not want to mix up with real code changes).